nx/test/nx/defn/grad_test.exs

```diff
@@ -1214,24 +1259,21 @@ defmodule Nx.Defn.GradTest do
     test "works with window max, no padding no stride" do
       x = Nx.iota({2, 3, 3, 2}, type: {:f, 32})
       lhs = grad_sum_window_max(x)
-      rhs = Nx.tensor([[[[0.0, 0.0],
-                         [0.0, 0.0],
-                         [0.0, 0.0]],
-                        [[0.0, 1.0],
-                         [0.0, 1.0],
-                         [0.0, 1.0]],
-                        [[0.0, 1.0],
-                         [0.0, 1.0],
-                         [0.0, 1.0]]],
-                       [[[0.0, 0.0],
-                         [0.0, 0.0],
-                         [0.0, 0.0]],
-                        [[0.0, 1.0],
-                         [0.0, 1.0],
-                         [0.0, 1.0]],
-                        [[0.0, 1.0],
-                         [0.0, 1.0],
-                         [0.0, 1.0]]]])
+
+      rhs =
```

Comment:
Should I revert this formatting?

Reply:
No, keep it formatted like you just did :)